sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -891,6 +1003,11 @@ public virtual async Task StopProcessingAsync(CancellationToken cancellationToke
                 {
                     await receiver.AbandonAsync(message.LockToken).ConfigureAwait(false);
                 }
+
```

Comment:
nit: can we add a comment explaining that we need propagate this in order to dispose the session receiver in the same place where are creating them.